src/services/SQSService.js

```diff
@@ -49,13 +51,16 @@ class SQSService {
         .findOne({ _id: mongoose.Types.ObjectId(type.template_id) })
         .exec();
 
-      const generatedTemplate = Mustache.render(template, { message: JSON.parse(event.Records[0].body).text });
-      const now = new Date();
+      const generatedTemplate = Mustache.render(template.template, { message: JSON.parse(event.Records[0].body).text });
```

Comment:
This doesn't seem right. What will JSON.parse(event.Records[0].body).text return? And why are you passing it into the render() method in a message field?

Reply:
done